src/tools/tidy/src/style.rs

```diff
@@ -39,6 +39,24 @@ C++ code used llvm_unreachable, which triggers undefined behavior
 when executed when assertions are disabled.
 Use llvm::report_fatal_error for increased robustness.";
 
+const ANNOTATIONS_TO_IGNORE: &[&str] = &[
+    "// @ has",
```

Comment:
I would rather not ignore this; I use this as 'I mean to add this test at some point but I don't want to make it a separate file', but others might not. `htmldocck` doesn't treat it specially.